sys-apps/shadow/files/pam.d-include/shadow

```diff
@@ -1,7 +1,7 @@
 #%PAM-1.0 
 
 auth       sufficient	pam_rootok.so
-auth       required		pam_permit.so
+auth       required		pam_deny.so
```

Comment:
I'd go without deny here, just drop the permit

Reply:
Dropped.